boms/cloud-lts-bom/pom.xml

```diff
@@ -267,8 +268,17 @@
       </dependency>
       <dependency>
         <groupId>com.google.cloud</groupId>
-        <artifactId>google-iam-admin</artifactId>
+        <artifactId>google-iam-admin-bom</artifactId>
```

Comment:
Using the BOM to cover API surface.

Reply:
Sorry, just to clarify, what's the reason for putting bom here instead of standard dependencies?

Reply:
It's to cover API surface.